HomeWork9/HomeWork9/Action.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Abstracts;
+using Models.Enum;
+using Services.Abstracts;
+
+namespace HomeWork9
+{
+    internal class Action : IAction
```

Comment:
ActionService

Reply:
it`s was task 
It is necessary to create the Actions class 
i think need name Action)